l1-contracts/contracts/interop/InteropCenter.sol

```diff
@@ -221,82 +169,45 @@ contract InteropCenter is IInteropCenter, ReentrancyGuard, Ownable2StepUpgradeab
             _bundleAttributes,
             AttributeParsingRestrictions.OnlyBundleAttributes
         );
+        // Verify that the unbundler address is not set to 0, to verify that sender didn't forget it.
+        // We want it to be set by sender because it's an important safety feature to have an option to unbundle the bundle.
+        require(bundleAttributes.unbundlerAddress != address(0), UnbundlerAddressZero());
```

Comment:
It might make sense to have a default unbundler address here. I.e. use interopAddress chainId address for unbundling, and have the unbundler be the original sender from the source chain. Add receiveMessage to the InteropHandler, to allow unbundling

Reply:
This will probably lead to reentrance, no?